nixos/tests/lorri/default.nix

```diff
@@ -0,0 +1,26 @@
+import ../make-test-python.nix {
```

Comment:
The VM test doesn't pass, as the python test runner didn't exist in 19.09 yet. However, porting that to the perl code should be straightforward - can you port testScript to perl?